doc/versionHistory.rst

```diff
@@ -6,6 +6,18 @@
 Version History
 ##################
 
+.. _lsst.ts.wep-14.16.0:
+
+-------------
+ 14.16.0
+-------------
+
+* Add neural network-based Zernike estimation task (CalcZernikesNeuralTask).
+* Integrate TARTS system for wavefront estimation.
+* Add support for WaveNet, AlignNet, and AggregatorNet models.
+* Fix type annotations throughout the codebase for better MyPy compliance.
+* Clean up linting issues and improve code quality.
```

Comment:
i think these last two lines are really not needed, as it's not as if you were fixing previous code but adding some new code, so it is expected that you will mypy it and lint it. Even if your commit history shows those commits in which you fix the formatting now, you will have to squash the commits and clean the git history before merging